### ISSUE TITLE
Unmounting doesn't work in IE

### ISSUE THREAD

**Issue by jgogstad** [evenchange4/react-grecaptcha#38]
When unmounting the recaptcha in IE, an exception is thrown `Object doesn't support property or method 'remove'`, due to https://github.com/evenchange4/react-grecaptcha/blob/master/src/Recaptcha.js#L48.  Which of course is because `remove` isn't available in IE.

**Comment by evenchange4** [comment]
How about just adding an `Element.prototype.remove()` polyfill inside your project?
So that `react-grecaptcha` will not polluting consumer's global environment.

```js
if (!('remove' in Element.prototype)) {
    Element.prototype.remove = function() {
        if (this.parentNode) {
            this.parentNode.removeChild(this);
        }
    };
}
```

ref: https://developer.mozilla.org/en-US/docs/Web/API/ChildNode/remove

Or, maybe we should use [babel-plugin-transform-runtime](https://babeljs.io/docs/plugins/transform-runtime/) in `react-grecaptcha`.

**Comment by jcdekoning** [comment]
What about changing the code to?

```
var scriptElelement = document.getElementById(ID);
scriptElement.parentNode.removeChild(scriptElement);
```

**Comment by evenchange4** [comment]
@jcdekoning Thanks! 🙌
Sorry I have no IE in my MacBook, please verify the released `v1.1.2` so that I could close it. cc @jgogstad

**Comment by evenchange4** [comment]
Close it. Please feel free to reopen this bug if the problem still occurs with a newer version.
